capstone/src/test/java/IsTooOldFunctionTest.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps;
+
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.Date;
+
+public class IsTooOldFunctionTest {
+    @Test
+    public void testIsTooOld1() throws Throwable {
+        Date date1 = new Date(2020, 1, 1);
+        Date date2 = new Date(2020, 6, 1);
+        DeleteFlags underTest = new DeleteFlags();
+        boolean result = underTest.isTooOld(date1, date2);
+        Assert.assertEquals(true, result);
```

Comment:
Move this call directly into the assertEquals. It's short and reads just like English inside an assert.

Reply:
changed

Reply:
No, I mean Assert.assertEquals(true, underTest.isTooOld(date1, date2));

Or better yet Assert.assertTrue(underTest.isTooOld(date1, date2));

It's very easy to see what it's doing because it reads like English in this form.

Reply:
Oh sorry, I commented "changed" but I haven't pushed my code yet.